orc8r/gateway/python/python.mk

```diff
@@ -113,10 +113,10 @@ $(PROTO_LIST): %_protos:
 	@echo "Generating python code for $* .proto files"
 	@mkdir -p $(PYTHON_BUILD)/gen
 	@echo "$(PYTHON_BUILD)/gen" > $(SITE_PACKAGES_DIR)/magma_gen.pth
-	$(BIN)/python $(SRC)/protos/gen_protos.py $(SRC)/$*/protos/ $(MAGMA_ROOT),$(MAGMA_ROOT)/orc8r/protos/prometheus $(SRC) $(PYTHON_BUILD)/gen/
+	$(BIN)/python$(PYTHON_VERSION) $(SRC)/protos/gen_protos.py $(SRC)/$*/protos/ $(MAGMA_ROOT),$(MAGMA_ROOT)/orc8r/protos/prometheus $(SRC) $(PYTHON_BUILD)/gen/
```

Comment:
Are the `$(PYTHON_VERSION)` changes necessary? I think `python` should already link to `python3.8`?

Reply:
So you think the tests were using Python 2 before? Would be surprising. Or do they run in a virtualenv?

It's probably safe to change in any case.